src/osx/core/colour.cpp

```diff
@@ -13,127 +13,244 @@
 #include "wx/colour.h"
 
 #ifndef WX_PRECOMP
-    #include "wx/gdicmn.h"
+#include "wx/gdicmn.h"
 #endif
 
 #include "wx/osx/private.h"
 
+CGColorSpaceRef wxMacGetGenericRGBColorSpace();
+
+wxColorRefData::~wxColorRefData()
```

Comment:
Do we actually still need defining empty dtors non-inline in wxMac? I thought this limitation didn't exist any longer.

Reply:
ok, I wasn't aware of that